packages/main/src/Link.ts

```diff
@@ -260,6 +298,20 @@ class Link extends UI5Element implements ITabbable {
 			&& this._dummyAnchor.protocol === loc.protocol);
 	}
 
+	_ensureNoIconOnlyLink() {
+		if (this._isIconOnly) {
+			throw new Error("Icon only links are not supported.");
+		}
+	}
+
+	get _isIconOnly() {
```

Comment:
As this getter is used on only one place (_ensureNoIconOnlyLink), probably you can move logic there and remove the getter.

Reply:
At all, can we state this in  the documentation and not adding additional JavaScript to handle the case?